### ISSUE TITLE
Sync the "haven't used remote-wsl" extension notification

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-remote-release#5932]
After setting up a new PC I get this:

![image](https://user-images.githubusercontent.com/2193314/142871916-26ef5e13-5f8e-46c3-9683-8663a12cf0dd.png)

I have used it a lot already, we should sync the flag that tracks whether it's been used.

**Comment by aeschli** [comment]
Interesting. I'm not yet sure how that can happen. We store two values in the same global store:
- a timestamp the first time we see that the Remote-WSL extension is installed
- a boolean that is set to true when a remote window is opened (or the user has seen the dialog)

Both are in the same storage which is one file (e.g. C:\Users\martinae\AppData\Roaming\code-oss-dev\User\globalStorage)

**Comment by Tyriar** [comment]
@aeschli is the boolean synced via settings sync?

**Comment by aeschli** [comment]
It shouldn't sync, as I do not call `Memento.setKeysForSync`

@sandy081 We don't sync the extension globalState by default, correct?

**Comment by Tyriar** [comment]
We should sync this imo, otherwise I get that notification whenever I set up a new machine (which I did twice last week)

**Comment by sandy081** [comment]
> We don't sync the extension globalState by default, correct?

Yes, we don't unless you set the key to sync using `Memento.setKeysForSync`

**Comment by aeschli** [comment]
Marking 'Sync the "haven't used remote-wsl" extension notification' as a feature request.
I know it doesn't explain the issue you were seeing...

**Comment by Tyriar** [comment]
@aeschli is this just a one liner to add `setKeysForSync`? If so, could it get done for December so we can close this out?

> I know it doesn't explain the issue you were seeing...

Not sure what you mean?

**Comment by aeschli** [comment]
I wanted to say that I have no explanation why, on a fresh install, there can be a warning about the extension being installed but not used given that we don't sync the global state this is relying on.

**Comment by joyceerhl** [comment]
I just got this notification right after installing the Remote WSL extension in VS Code stable. Is this notification really necessary (especially right after installing the extension)?
